packages/blade/src/components/Chip/AnimatedChip.web.tsx

```diff
@@ -14,7 +14,9 @@ const AnimatedChip = styled(BaseBox)<AnimatedChipProps>((props) => {
   );
   return {
     ...getAnimatedChipStyles(props),
-    width: 'fit-content',
+    width: props?.width ? (props.width as string | number) : 'fit-content',
+    maxWidth: props?.maxWidth as string | number,
+    minWidth: props?.minWidth as string | number,
```

Comment:
why is `as string  | number` required here? isn't styled-components by default accepting string and number?